TestSolutions/Lombiq.Analyzers.PackageReference/IndirectPackageReference/IndirectPackageReference.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <!-- This project demonstrates that the package reference doesn't transfer the analyzer effect to its dependents. If
+       you set ImportPackageAgain to "true" and run static code analysis on this soltuion you will get twice as many
+       warnings. This is the desired behavior, shows that the analyzers only activate when they are explicitly imported.
```

Comment:
I seem to be missing something. I don't see any indirect reference to the Analyzers here.

Reply:
Indirect package reference means that this project references Lombiq.Analyzers.PackageReference which in turn references Lombiq.Analyzers. In other words we show that just because you import a project that uses Lombiq.Analyzers your project won't get "infected" by Lombiq.Analyzers against your will. Only directly importing Lombiq.Analyzers activates its effects.

Reply:
WTF, it vanished!! 😱

Reply:
Ok, now it's fixed. Updated related documentation too.